app/models/ingredient.rb

```diff
@@ -0,0 +1,6 @@
+class Ingredient < ApplicationRecord
+  belongs_to :recipe
+  attribute :name, :string
+  attribute :quantity, :integer
+  attribute :value, :decimal
+end
```

Comment:
/- [OPTIONAL] Kindly add input validation to validate the Ingredient records to prevent the user from submitting an employee field or unrelated data to your database. ✍🏻